ovos-i2csound

```diff
@@ -38,8 +38,52 @@ detect_i2c_device() {
     fi
 }
 
+pulseaudio_config() {
+    echo "configuring pulseaudio"
+    # Set default configurations
+    if [ -f /var/lib/alsa/default-asound.state ]; then
+        ASOUND_STATE=/var/lib/alsa/default-asound.state
+    fi
+    if [ -f /etc/pulse/system.pa.d/pulseaudio-system.pa ]; then
+        PULSE_SYSTEM=/etc/pulse/system.pa.d/pulseaudio-system.pa
+    fi
+    if [ -f /etc/pulse/daemon.conf.d/pulseaudio-daemon.conf ]; then
+        PULSE_DAEMON=/etc/pulse/daemon.conf.d/pulseaudio-daemon.conf
+    fi
+
+    # Remove old configurations
+    if [ -f /etc/pulse/system.pa ] || [ -L /etc/pulse/system.pa ]; then
+        rm /etc/pulse/system.pa
```

Comment:
Don't do this if you are using drop-in files. It is not needed and because the other files are now within the ".d" folders, this will actually prevent those drop-in files to be loaded as those files contain this at the bottom, for instance;

```
### Allow including a default.pa.d directory, which if present, can be used
### for additional configuration snippets.
### Note that those snippet files must have a .pa file extension, not .conf
.nofail
.include @PA_DEFAULT_CONFIG_DIR_UNQUOTED@/default.pa.d
```

Reply:
I did not know this.  Thanks